tests/test_cmt.py

```diff
@@ -9,12 +9,13 @@
 from .test_plugins import test_run_id_nT
 from straxen.common import aux_repo
 
+
 def test_connect_to_db():
     """
     Test connection to db
     """
     if not straxen.utilix_is_configured():
-        warn('Cannot do test becaus  '
+        warn('Cannot do test because  '
              'no have access to the database.')
         return

```

Comment:
I am wondering if we should not make this whole thing a utility function. We require it for quite a many tests.

Reply:
Thanks, I think that makes sense, want to check if my changes are similar to what you suggested?